packages/openapi3/src/schema-emitter.ts

```diff
@@ -645,6 +651,63 @@ export class OpenAPI3SchemaEmitter extends TypeEmitter<
     }
   }
 
+  #attachXmlObject(program: Program, prop: ModelProperty, emitObject: OpenAPI3Schema) {
+    const xmlObject: OpenAPI3XmlSchema = {};
```

Comment:
when a model contains multiple xml decorators for model properties, looks like the xml object is created multiple times when process a model property, and previous existing xml object's property will be deleted.
try test this model:
```
model Pet {
  @encodedName("application/xml", "ItemsTags")
  @xml.unwrapped
  tags: Tag[];
@Xml.attribute
  id: string;
}
```
so, we should check if `emitObject` has `xml`, if yes, reuse it.